compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -122,6 +122,13 @@ public void visit(BLangSimpleVarRef varRef) {
             return;
         }
 
+        if (!this.unresolvedConstants.containsKey(varRef.symbol)) {
+            dlog.error(varRef.pos, DiagnosticErrorCode.INVALID_CONST_EXPRESSION,
+                    "cannot resolve '" + varRef.symbol.name.value + "'");
```

Comment:
Let's add a new diagnostic code (like CANNOT_RESOLVE_CONST) for this and pass the name of the constant as an argument.

e.g., https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java#L38